src/devtools/client/debugger/src/actions/pause/commands.js

```diff
@@ -67,59 +45,26 @@ export function stepOver(cx) {
   };
 }
 
-/**
- * stepOut
- * @memberof actions/pause
- * @static
- * @returns {Function} {@link command}
- */
-export function stepOut(cx) {
+export function reverseStepOver(cx) {
   return dispatch => {
     if (cx.isPaused) {
-      return dispatch(command(cx, "stepOut"));
+      return dispatch(command(cx, "reverseStepOver"));
     }
   };
 }
 
-/**
- * resume
- * @memberof actions/pause
- * @static
- * @returns {Function} {@link command}
- */
-export function resume(cx) {
+export function stepOut(cx) {
   return dispatch => {
     if (cx.isPaused) {
-      recordEvent("continue");
-      return dispatch(command(cx, "resume"));
+      return dispatch(command(cx, "stepOut"));
     }
   };
 }
 
-/**
- * rewind
- * @memberof actions/pause
- * @static
- * @returns {Function} {@link command}
- */
-export function rewind(cx) {
-  return dispatch => {
-    if (cx.isPaused) {
-      return dispatch(command(cx, "rewind"));
-    }
-  };
+export function resume(cx) {
+  return dispatch => dispatch(command(cx, "resume"));
 }
 
-/**
- * reverseStepOver
- * @memberof actions/pause
- * @static
- * @returns {Function} {@link command}
- */
-export function reverseStepOver(cx) {
-  return dispatch => {
-    if (cx.isPaused) {
-      return dispatch(command(cx, "reverseStepOver"));
-    }
-  };
+export function rewind(cx) {
+  return dispatch => dispatch(command(cx, "rewind"));
```

Comment:
It seems odd to both ensure that `isPaused` is true and also to delete the place where we check it?